cost/azure/old_snapshots/azure_delete_old_snapshots.pt

```diff
@@ -461,15 +461,13 @@ policy "pol_azure_delete_old_snapshots" do
 EOS
```

Comment:
Can see this policy is missing the comment separators for each section e.g., 
`###############################################################################
# Policy
###############################################################################`
and
`###############################################################################
# Escalations
###############################################################################`
worth adding these in? Not sure whether this applies to other policies in this PR too